simple-jwt-login/src/Services/BaseService.php

```diff
@@ -162,7 +162,7 @@ public function getJwtFromRequestHeaderOrCookie()
         if ($this->jwtSettings->getGeneralSettings()->isJwtFromHeaderEnabled()) {
             $headers = array_change_key_case($this->serverHelper->getHeaders(), CASE_LOWER);
             $headerKey = strtolower($this->jwtSettings->getGeneralSettings()->getRequestKeyHeader());
-            if (isset($headers[$headerKey])) {
+            if (isset($headers[$headerKey]) && !empty(trim($headers[$headerKey]))) {
```

Comment:
First of all, thank you for your contribuiton to this plugin.

I've checked then code, and It seems it does not fix the issue.
If the header is "Bearer " then `!empty("Bearer ")` will be true.

This check should be done after we extract the token from the header( line 173).

Also, it would be nice to cover this scenario with unit tests.